ecs-cluster/keycloak.tf

```diff
@@ -100,7 +104,7 @@ resource "aws_alb_target_group" "keycloak" {
     protocol            = "HTTPS"
     matcher             = "200"
     timeout             = "5"
-    path                = "/"
+    path                = "/health"
```

Comment:
Do we need to update the healthcheck?

https://www.keycloak.org/server/health
suggests it should be
`/health/ready` or `/health/live`

Reply:
Previously Keycloak healthchecks were disabled, and `/` returned HTTP status 200 which was fine. In the current version `/` now redirects. I went with `/health` because it seems to work fine https://quarkus.io/guides/smallrye-health#running-the-health-check